repos/fdbt-site/src/pages/api/capConfirmation.ts

```diff
@@ -0,0 +1,12 @@
+import { NextApiResponse } from 'next';
+import { redirectToError, redirectTo } from '../../utils/apiUtils';
+
+export default (res: NextApiResponse): void => {
```

Comment:
could we include req here also? but like this 
```suggestion
export default (_req: NextApiRequestWithSession, res: NextApiResponse): void => {
```